### ISSUE TITLE
Align clarity-ui and clarity-core colors

### ISSUE THREAD

**Issue by mathisscott** [vmware/clarity#4546]
## Summary

_Clarity Core had to make changes to the colors in tags and labels to support the WCAG 2.1 guidelines for graphical elements. These changes should be replicated in clarity-ui for consistency and accessbility._

- What is the change?

See the following PR #4499 

### General
- [x] the warning-800 needs to change from `hsl(41, 100%, 36%)` to `hsl(41, 100%, 34%)`

### Badges
- [x] orange badge bg-color changes from 500 to 1000
- [x] orange badge text color changes from 1000 to 0
- [x] light-blue badge bg-color changes from 300 to 500
- [x] light-blue badge text color changes from 1000 to 0
- [x] info badge bg-color changes from 600 to 800
- [x] warning badge bg-color changes from 300 to 1000
- [x] success badge bg-color changes from 700 to 800
- [x] danger badge bg-color changes from 800 to 900

### Labels
Note that non-status labels use the `bg-color` variables for border colors. This is a problem with the legacy code.

- [x] the gray label text color should be 1000, not 700
- [x] the purple label text color should be 1000, not 700
- [x] the orange label bg-color (a.k.a. border color) should be 1000 not 500
- [x] the light-blue label bg-color (a.k.a. border color) should be 500 not 300
- [x] the info label border-color variable should be 800 not 400
- [x] the success label border-color variable should be 800 not 400
- [x] the warning label border-color variable should be 800 not 500
- [x] the danger label border-color variable should be 900 not 200

### Alerts

- [x] info alert border color should be 800, not 400
- [x] info alert icon color should be 800, not 400
- [x] success alert border color should be 800, not 400
- [x] success alert icon color should be 800, not 700
- [x] warning alert text color should be 900, not 700
- [x] warning alert border color should be 800, not 400
- [x] warning alert icon color should be warning-800, not neutral-700
- [x] danger alert border color should be 900, not 200
- [x] danger alert icon color should be 900, not 800
- [x] app-alert success background color should be 700, not 500

- Why should it go in Clarity?

Consistency with clr-core and alignment with WCAG 2.1.

- Does this change impact existing behaviors? If so how?

Only a visual change.

**Comment by gnomeontherun** [comment]
@mathisscott Update this to include the alert colors.

**Comment by hippee-lee** [comment]
Started this issue. I have a [branch](https://github.com/hippee-lee/clarity/tree/topic/4546-align-colors) that can incorporate the alert changes when they are ready.
